storage/mysql/log_storage.go

```diff
@@ -348,6 +350,7 @@ func (t *logTreeTX) QueueLeaves(ctx context.Context, leaves []*trillian.LogLeaf,
 		if len(leaf.LeafIdentityHash) != t.hashSizeBytes {
 			return nil, fmt.Errorf("queued leaf must have a leaf ID hash of length %d", t.hashSizeBytes)
 		}
+		leaf.QueueTimestampNanos = queueTimestamp.UnixNano()
```

Comment:
FYI, you'll probably need ptypes.TimestampProto here. (See comment on proto file below.)

https://godoc.org/github.com/golang/protobuf/ptypes#TimestampProto